### PR TITLE
Lesson flow update

### PR DESCRIPTION
**Pull request by jannatulabeda**
- Update lesson API implement in lesson viewcontroller
- Checkings added
- Result viewController update
- Fix lesson navigation flow

### CODE REVIEW

File: fels_111/ResultTableViewController.m [hunk 1]
```diff
@@ -44,58 +48,19 @@ - (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger
 
 - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath {
     ResultTableViewCell *cell = (ResultTableViewCell *)[tableView dequeueReusableCellWithIdentifier:@"resultCell" forIndexPath:indexPath];
-    // Configure the cell...
-    cell.resultImageView.image = [UIImage imageNamed:@"avatar.png"];
     Word *word = [self.resultArray objectAtIndex:indexPath.row];
+    // Set Image
+    if (word.isCorrectlyAnswered) {
```
Comment (TequilarTomsk) [inline fels_111/ResultTableViewController.m]: please define string for string [UIImage imageNamed:@"rightSign.png"]] and [UIImage imageNamed:@"crossSign.png"]]
And use Trinity operator

File: fels_111/CategoryTableViewController.m [hunk 2]
```diff
@@ -28,12 +28,11 @@ @implementation CategoryTableViewController
 
 - (void)viewDidLoad {
     [super viewDidLoad];
-    
     self.categoryArray = [[NSMutableArray alloc] init];
     // Initially page number is set to 1
     pageNumber = 1;
     // Number of data per page is set to 3 for testing purpose
-    perPageData = 3;
+    perPageData = 10;
```
Comment (TequilarTomsk) [inline fels_111/CategoryTableViewController.m]: why the number is hard-coded here?

Comment (jannatulabeda) [inline fels_111/CategoryTableViewController.m]: Should I define it in constant?

File: fels_111/ResultTableViewController.m (hunk 1 above)
Comment (jannatulabeda) [inline fels_111/ResultTableViewController.m]: Okay doing

**Pull request merged**